tests/mmrelay/test_meshtastic_utils.py

```
@@ -0,0 +1,1237 @@
+import sys
+import os
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
+
+import sys
```

Flake8 (7.2.0)

[error] module level import not at top of file

(E402)